### DISCUSSION TITLE
How can I open a modal from a route?

### DISCUSSION THREAD

**Discussion by AnderssonGt** [wire-elements/modal#324] [General]

**Comment by SfaygaTheKing** [comment]
Hey,
did u find the answer?

**Comment by PhiloNL** [comment]
It's not possible top open it directly from a route but you can do this by using a view:

```
// web.php
Route::view('foo/bar', 'open-modal')

// open-modal.blade.php
<script>
Livewire.dispatch('openModal', { component: 'foo-bar' })
</script>
```

**Comment by SfaygaTheKing** [comment]
hey, thanks for the reply, soo the thing is i am trying to do is after creating an object i send an email with a link to view the object created. The view function is treated with livewire modal that is trigged when i click on an action button in the table of objects. I want that link to take me to that page containing the table with the view modal opened. I tried different things but still no solution.
